scripts/test-ai-system.py

```
@@ -0,0 +1,123 @@
+                cost = 0.0
+                print(f"💰 Cost: ${cost} (free)")
+            else:
+                # Rough estimate
+                message_str = str(test_case["message"])
+                tokens = len(message_str.split()) * 1.3
+                cost = (tokens / 1000) * 0.0006
+                print(f"💰 Cost: ${cost:.6f} (estimated)")
+
+            print(f"✅ Test {i} completed successfully")
+
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)